src/EnergyPlus/DataZoneEquipment.cc

```diff
@@ -1481,8 +1477,8 @@ namespace EnergyPlus::DataZoneEquipment {
                 }
                 ZoneOAArea = Zone(ActualZoneNum).FloorArea * Zone(ActualZoneNum).Multiplier * Zone(ActualZoneNum).ListMultiplier *
                              OARequirements(DSOAPtr).OAFlowPerArea;
-                ZoneOAMin = ZoneOAArea / ZoneEz;
-                ZoneOAMax = (ZoneOAArea + ZoneOAPeople) / ZoneEz;
+                ZoneOAMin = ZoneOAArea;
+                ZoneOAMax = (ZoneOAArea + ZoneOAPeople);
```

Comment:
In this function `ZoneEz` was always 1.0. Remove it here and below. The real `ZoneEz` gets applied elsewhere.

Reply:
Good catch!